src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+@import "../utils/variables";
+
+.card {
+  width: 200px;
```

Comment:
Setting width as fixed is not the best idea most of the time. It would be better to set max-width and set width as 100%.

Reply:
Yes, you are rigth but requirements says that card width is `200px` including border